Documentation/General/finalReport/overview.tex

```diff
@@ -1 +1,40 @@
-\chapter{Overview of the developed and implemented software product}
\ No newline at end of file
+\chapter{Overview of the developed and implemented software product}
+
+\section{Introduction}
+When entering MoodCat, a user can choose multiple of a list of moods to listen music to. Based on these moods a list of rooms
```

Comment:
Friso has mentioned this several times actually. What is moodcat? I don't know, this is the place to tell me